release-notes/stable.yml

```diff
@@ -1,4 +1,10 @@
 releases:
+  39.20240225.3.0:
+    issues: []
+  39.20240210.3.0:
+    issues:
+      - text: "Using zram in live PXE causes services with PrivateDevices=yes to fail"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1296"
```

Comment:
Based on the last comment there, that might've been a false alarm. We should probably actually remove that item from the previous `next` and `testing` release notes too.

Reply:
Sounds good. I was trying to understand the comment and removal of the label.